lib/generator.js

```diff
@@ -17,13 +17,21 @@ TemplateGenerator.prototype.render = function render(mail, cb) {
     if (mail.data.html) return cb();
 
     var templatePath = path.join(this.viewPath, mail.data.template + this.extName);
-
+    var text_templatePath='';
```

Comment:
Please use camel casing when naming variables, you also don't need to initialize it, simply defining it will do.